ios/FluentUI/Table View/TableViewTokens.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+ import UIKit
+
+ open class TableViewTokens: ControlTokens {
+    //TODO: Update darkElevated value
+    open var backgroundColor: DynamicColor {
+        .init(light: globalTokens.neutralColors[.white],
+              dark: globalTokens.neutralColors[.black])
+    }
+
+     //TODO: Update backgroundGrouped for light/dark/darkElevated
+    open var backgroundGrouped: DynamicColor {
+        .init(light: aliasTokens.backgroundColors[.neutral2].light,
+              dark: aliasTokens.backgroundColors[.neutral1].dark)
+    }
+}
+
+ open class TableViewCellTokens: TableViewTokens {
+    open var cellBackgroundColor: DynamicColor { aliasTokens.backgroundColors[.neutral1] }
+
+     //TODO: Replace cellBackgroundGrouped with 2.0 token
```

Comment:
Is this part of the fluent2 color updates? I think this should be tracked in ADO instead of TODO items in the code so we can manage it in an easier way. (same question applies for the other TODO items in this file)

Reply:
This is indeed part of the new color updates, I will add tracking info to them